addons/openebs/1.6.0/Manifest

```diff
@@ -10,3 +10,6 @@ image node-disk-manager-amd64 quay.io/openebs/node-disk-manager-amd64:v0.4.6
 image node-disk-operator-amd64 quay.io/openebs/node-disk-operator-amd64:v0.4.6
 image openebs-k8s-provisioner quay.io/openebs/openebs-k8s-provisioner:1.6.0
 image provisioner-localpv quay.io/openebs/provisioner-localpv:1.6.0
+
+yum iscsi-initiator-utils
```

Comment:
This is a new capability for the addon Manifest. You can now specify yum and apt packages and the `bin/save-manifest-assets.sh` script will download the `rpm` and `dep` archives and include them in the addon script. The addon install script has to detect if/which archives need to be installed and install them with rpm or dpkg.